container-search/src/main/java/com/yahoo/search/rendering/JsonRenderer.java

```diff
@@ -317,6 +317,10 @@ protected void renderErrors(Set<ErrorMessage> errors) throws IOException {
         generator.writeEndArray();
     }
 
+    protected boolean shouldRenderStacktraceOf(Throwable cause) {
```

Comment:
either update abi-spec or make this private.  Also consider moving the
`cause.getStackTrace().length > 0` condition into `shouldRenderStacktraceOf()`

Reply:
Thanks, added. It makes sense to be able to override this in a subclass. For the same reason the existence check should be where it is.